openassessment/workflow/models.py

```diff
@@ -239,7 +240,8 @@ def get_score(self, assessment_requirements, step_for_name):
                     else:
                         requirements = assessment_requirements.get(assessment_step_name, {})
                     score = get_score_func(self.submission_uuid, requirements)
-                    break
+                    if score:
+                        break
```

Comment:
Hmm.. I may be missing something but the score is only set if all the steps have `assessment_completed_at` set. And that will only happen for the peer assessment step if the submission has been graded by the peers. But I'll read through the code again in the morning.

https://github.com/edx/edx-ora2/blob/usman/tnl1162-staff-assessment/openassessment/workflow/models.py#L322